### DISCUSSION TITLE
Using RubyMine for Fastlane Development

### DISCUSSION THREAD

**Discussion by hisaac** [fastlane/fastlane#17123] [Q&A]
Is anyone using RubyMine for working with their Fastfiles? I'd like to be able to do this, but am unsure how to configure it.

I can open the Fastfile in RubyMine just fine, but the most notable thing that's lacking is the ability to do introspection on Fastlane methods. All Fastlane methods show a warning saying, "Cannot find <method name>".

![Screen Shot 2020-09-01 at 11 14 24 AM](https://user-images.githubusercontent.com/923876/91877986-536ff980-ec44-11ea-9086-6e33acf8d63e.png)

If anyone has advice on how to configure RubyMine to successfully pick up the Fastlane commands, I'd greatly appreciate it!

**Comment by hongfeiyang** [comment]
Has someone figured it out ? After adding a Gemfile how to make the IDE understand those built-in fastlane action functions
